storage/mysql/mysql.go

```diff
@@ -159,3 +182,126 @@ func (s *Storage) ReadEntryBundle(ctx context.Context, index uint64) ([]byte, er
 	var entryBundle []byte
 	return entryBundle, row.Scan(&entryBundle)
 }
+
+// Add is the entrypoint for adding entries to a sequencing log.
+func (s *Storage) Add(ctx context.Context, entry *tessera.Entry) (uint64, error) {
+	// TODO: Return index if the value is already stored.
```

Comment:
nit: TODOs should be either `TODO(username)` or `TODO(issueid)`. We should set up a presubmit validator for this...

Reply:
Linked it back to #21, and I've added a task list in #21.